build.gradle

```diff
@@ -1999,10 +1999,14 @@ project(':tools') {
     implementation project(':clients')
     implementation project(':storage')
     implementation project(':server-common')
+    implementation project(':server')
```

Comment:
Is this dependency necessary? we don't expect tool module depends on core module.

Reply:
Great catch. Thanks!
It seems this dependency not required. Removed.